packages/puppeteer-core/src/api/ElementHandle.ts

```diff
@@ -626,9 +626,21 @@ export abstract class ElementHandle<
   /**
    * Returns the middle point within an element unless a specific offset is provided.
    */
-  async clickablePoint(offset?: Offset): Promise<Point>;
-  async clickablePoint(): Promise<Point> {
-    throw new Error('Not implemented');
+  async clickablePoint(offset?: Offset): Promise<Point> {
+    const model = await this.boundingBox();
```

Comment:
I do not think it's equivalent to the previous implementation. In particular, it would have a different behavior for elements with multiple bounding boxes. Could you please elaborate what where the features of the previous implementation and how the new implementation covers them?